src/core/mac/mac_frame.hpp

```diff
@@ -694,7 +694,7 @@ class Frame: public RadioPacket
     uint8_t *FindSrcPanId(void);
     uint8_t *FindSrcAddr(void);
     uint8_t *FindSecurityHeader(void);
```

Comment:
As far as I know, there is no way to annotation pointer returns like this to indicate how long they can be read from. I'd suggest we refactor the code that uses these. Another reason I'd suggest a refactor is that it in many places, it seems the Mac Header ends up getting read multiple times because of these helper functions.